lib/Pipfile.locks/python-2.7.16

```diff
@@ -21,6 +21,13 @@
             ],
             "version": "==0.26.2"
         },
+        "astor": {
```

Comment:
what did it change here?

Reply:
Not sure exactly what's going on with the diff here actually